spec/request/institutions_request_spec.rb

```diff
@@ -27,7 +27,8 @@
     end
 
     it 'allow searching by address fields' do
```

Comment:
Can we get a spec showing that it does not return institutions where `approved: false`?

Reply:
:heavy_plus_sign:

Reply:
Added the approved: false check to existing specs.